### DISCUSSION TITLE
GitHub Action Workflow cleanup v0.19.0 🌈

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#934] [Q&A] [answered]
## Changes

- feat: Organize GitHub Actions ci.yml @bdougie (#933)

<hr /><em>This discussion was created from the release <a href='https://github.com/open-sauced/open-sauced/releases/tag/v0.18.5'>GitHub Action Workflow cleanup v0.19.0 🌈</a>.</em>

**Comment by 0-vortex** [comment]
There is a small improvement for running release on top #1064 that is running release on top of the previous build step, can hold the PR back and do this as part of that if you think it's worth it :D

**Comment by 0-vortex** [comment, marked as answer]
marking [@open-sauced/semantic-release-conventional-config](https://github.com/open-sauced/semantic-release-conventional-config) as de-facto answer to releasing :D
